timesheet_automatic_remind/__openerp__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# © 2015 Elico corp (www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Timesheet Automatic Remind',
+    'version': '8.0.1.0.3',
+    'category': 'Human Resources',
+    'depends': [
+        'hr_timesheet_sheet',
+        'project',
+        'hr_holidays',

```

Comment:
I mean hr_holidays_compute_days is already depending on hr_holidays right: do you really need to redeclare the dependency?